apps/server/src/shared/domain/entity/task.entity.ts

```diff
@@ -85,6 +86,10 @@ export class Task extends BaseEntityWithTimestamps implements ILearnroomElement,
 	@OneToMany('Submission', 'task')
 	submissions = new Collection<Submission>(this);
 
+	@Index()
+	@ManyToOne('TaskCard', { fieldName: 'taskCardId', nullable: true })
```

Comment:
Why we set ManyToOne and not OneToOne? 
the task-card has one-to-one with task, so setting it here differently, I think might lead to problems.

I was thinking no entity relationship at all, only storing a string. This would have no performance impact at all.
 One other question I had was if we must define the relation on both entities. Or define it as inverse side here.

Reply:
@virgilchiriac implemented as string property instead of linked entity